osbuild/buildroot.py

```diff
@@ -258,9 +259,11 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None):
             api_path = "/run/osbuild/api/" + api.endpoint
             mounts += ["--bind", api.socket_address, api_path]
 
+        bwrap_caps = ["CAP_SYS_ADMIN", "CAP_SYS_CHROOT", "CAP_NET_ADMIN", "CAP_SETUID", "CAP_SETGID", "CAP_SYS_PTRACE"]
+        drop_caps = ", ".join(set(bwrap_caps).symmetric_difference(set(self.capabilities)))
+
         cmd = [
             "bwrap",
-            "--cap-add", "CAP_MAC_ADMIN",
```

Comment:
Where did that go? Some stages need `CAP_MAC_ADMIN` ;)

Reply:
I have added it back :grin: